### ISSUE TITLE
Buffer implementation inconsistent with NodeJS - Bun outputs array buffer

### ISSUE THREAD

**Issue by aniketbiprojit** [oven-sh/bun#930]
`.toString` implementation is seems flawed or at least inconsistent with NodeJS:

### Test.ts
```test.ts
import { readFileSync } from 'fs'

const completePath = './data'

const data = readFileSync(completePath).toString()

console.log({ data: data.toString() })
``` 

### Outputs:

```
⋊> ts-node ./test.ts
{ data: 'data' }


⋊> bun test.ts
{ data: "100,97,116,97" }

⋊> npx tsc test.ts
⋊> node test.js
{ data: 'data' }
```

**Comment by aniketbiprojit** [comment]
#155

**Comment by Electroid** [comment]
Yes we will fix this, tracking the issue in #155
